buildkite/pipeline.yml

```diff
@@ -17,3 +17,7 @@ steps:
 
   - label: ":shipit: Push images"
     command: docker/push
+
+notify:
```

Comment:
can you investigate if these are safe to leave in public code? if not I think that buildkite does environment variable interpolation, so we could redact the token part of this (between `workflows` and `triggers`) and replace with an token that we provision into the machine - and then revoke this one